cmake/AwsCFlags.cmake

```diff
@@ -125,6 +125,8 @@ function(aws_set_common_properties target)
         if (NOT MSVC)
             list(APPEND AWS_C_FLAGS "-fvisibility=hidden")
         endif()
+
+        set_target_properties(${target} PROPERTIES VERSION 1.0.0 SOVERSION 0)
```

Comment:
Why are we forcing everyone who uses our CFlags to be at version 1.0.0? Won't we want to release at different paces?

Reply:
I was making sure there's at least something set here. If we don't do this, there will not be a so.version output unless the consuming project hand specifies VERSION on their target.

Reply:
I think we punt this. I'd rather it didn't work and someone have to manually set their version than just go "ah but it works, I don't have to set this."